CMakeLists.txt

```
@@ -37,7 +37,8 @@ OPTION(BUILD_TESTS "xparrow test suite" OFF)
 # =====
 
 set(XPARROW_HEADERS
```

Not related to this PR: currently headers aren't installed because of difference in case.

This line must be change to:
```suggestion
set(Xparrow_HEADERS
```

or `Xparrow_HEADERS` bellow must be changed to `XPARROW_HEADERS`.